webapp/src/js/app.js

```diff
@@ -80,7 +80,7 @@ _.templateSettings = {
     $urlRouterProvider.otherwise('/error/404');
     router($stateProvider);
     $urlRouterProvider.when('', '/home');
-    $urlRouterProvider.when('/messages/{uuid}', '/messages/contact:{uuid}');
+    $urlRouterProvider.when('/messages/{uuid:[^:]*}', '/messages/contact:{uuid}');
```

Comment:
@garethbowen I noticed you addressed this problem differently in your branch [here](https://github.com/medic/medic-webapp/commit/1faf1ca31150fa8d3b9b50234a919953517adce8#diff-0f9e6b88aa7ac5a3d32e1eb29edbc904R78). I'm not entirely sure on the usecases here, but obviously my version won't work if the uuid can contain a `:` (which I assume it can't), but happy to use your version if there's something else I'm missing.

Reply:
Your solution looks much better. Good one.